_docs_integrate/delete-an-attribute.md

```diff
@@ -0,0 +1,127 @@
+---
+# !!! Warning: Do not edit this file; any changes must be replicated in Excel !!!
+permalink: integrate/delete-an-attribute
+published: true
+title: "Delete an Attribute"
+type: scenario
+toc: true
+properties:
+  - id: SC055
+  - category: Handling data deletion
+  - description:
+  - customer: All
+  - component: integrate
+  - level: Advanced
+  - implementation status: OPEN
+  - documentation status: OPEN
+  - published:
+  - link: integrate/delete-an-attribute
+require:
+required_by:
+---
+
+The exact process of deleting a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) depends on the kind of LocalAttribute at hand.
+Besides creating [IdentityAttributes]({% link _docs_integrate/data-model-overview.md %}#identityattribute) for yourself, a common usage of enmeshed is to share them with a peer or to receive shared Attributes from a peer.
+These different cases have different demands that need to be taken into account, wanting to delete an Attribute.
+
+In general, you can only delete data from your own wallet and never from the peer's.
+So, you can delete peer shared Attributes, your copy of own shared Attributes and unshared RepositoryAttributes.
+However, if you want the peer to delete an Attribute you shared with them, you will need to request it, since the peer may have a valid reason to still keep it for a certain amount of time.
+
+## Request the deletion of own Attributes from peer
+
+Wanting to delete an Attribute owned by you from the peer technically describes the endeavor of reverting sharing the Attribute with them.
+To this end, a [Request]({% link _docs_integrate/data-model-overview.md %}#request) must be used with a [DeleteAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#deleteattributerequestitem).
+As a parameter, the `attributeId` of the shared Attribute you would like the peer to delete must be provided.
```

Comment:
I think it depends, so if you refer to both own shared Attributes and peer shared Attributes at the same time, I think there are cases where it would be reasonable to talk just about 'shared Attributes'. However, when it comes to deleting peer shared Attributes, I'd write that the `id` of the 'peer shared Attibute' and not the 'shared Attribute' is required, because then it is not about own shared Attributes, even though the `id` of a peer shared Attribute and the `id` of its corresponding own shared Attribute are the same. I think the way it is written now (with the remark that the `id` of a peer shared Attribute and the `id` of its corresponding own shared Attribute are the same) is great.